packages/hint-button-type/README.md

```diff
@@ -0,0 +1,62 @@
+# button-type (`@hint/hint-button-type`)
+
+Check if button has type attribute set
+
+## Why is this important?
+
+The default type for <button> is "submit" (not type="button" as one might expect). This can lead to surprising keyboard behavior within a form. 
```

Comment:
```suggestion
The default type for `<button>` is `"submit"` (not `type="button"` as one might expect). This can lead to surprising keyboard behavior within a form. 
```